dashboard/src/components/EventsContainer.js

```diff
@@ -14,17 +14,14 @@ import EventsContainerMenu from "./EventsContainerMenu";
 
 const styles = theme => ({
   eventsContainer: {
-    border: "1px solid",
-    borderRadius: 2,
-    borderColor: theme.palette.divider,
     marginTop: 16,
     marginBottom: 16,
+    backgroundColor: theme.palette.background.paper,
```

Comment:
Occurred to me that the events container should probably be a Paper element as well. Unrelated to this PR so it can be handled separately though. 🤔